src/paperless/settings.py

```diff
@@ -46,7 +48,9 @@ def __get_boolean(key: str, default: str = "NO") -> bool:
     Return a boolean value based on whatever the user has supplied in the
     environment based on whether the value "looks like" it's True or not.
     """
-    return bool(os.getenv(key, default).lower() in ("yes", "y", "1", "t", "true"))
+    return bool(
```

Comment:
I'm not loving how many unchanged lines have changed in this diff. It appears a bunch of commas were added. It makes it difficult to review. Could you please remove these changes?